src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -564,10 +621,30 @@ namespace winrt::TerminalApp::implementation
             // Do nothing if the Tab's lifetime is expired or pane isn't new.
             auto tab{ weakThis.get() };
 
-            if (tab && sender != tab->_activePane)
+            if (tab)
             {
-                tab->_UpdateActivePane(sender);
-                tab->_RecalculateAndApplyTabColor();
+                if (sender != tab->_activePane)
+                {
+                    tab->_UpdateActivePane(sender);
+                    tab->_RecalculateAndApplyTabColor();
+                }
+                tab->_focusState = WUX::FocusState::Programmatic;

```

Comment:
i hope this is right! what else is `_focusState` being used for?

Reply:
The only other places (before this PR) `_focusState` was being checked/updated: `TerminalTab::_RefreshVisualState` and `TerminalTab::Focus`

However, without this addition, the tab does not know when the app itself loses focus (by alt-tab/minimize etc) so it used to still think it is focused even though the app itself _and_ the underlying control(s) are all unfocused.